src/main/java/christmas/ui/Converter.java

```diff
@@ -0,0 +1,52 @@
+package christmas.ui;
+
+import static christmas.domain.Date.DATE_RE_READ_REQUEST_MESSAGE;
+import static christmas.domain.OrderedMenu.ORDER_RE_READ_REQUEST_MESSAGE;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Converter {
+    private static final String ORDER_DELIMITER = ",";
+    private static final String NAME_AND_AMOUNT_DELIMITER = "-";
+
+    public static int convertDateNumeric(String Input) {
+        try {
+            return Integer.parseInt(Input);
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(DATE_RE_READ_REQUEST_MESSAGE);
+        }
+    }
+
+    public static List<String> separateOrder(String order) {
+        return new ArrayList<>(List.of(order.split(ORDER_DELIMITER)));
```

Comment:
(이렇게도 가능해요!)

ArrayList로 다시 감싸지 않아도 충분할 것 같아요 :)
```suggestion
          return List.of(order.split(ORDER_DELIMITER));
```